sky/clouds/service_catalog/data_fetchers/fetch_gcp_metadata.py

```diff
@@ -0,0 +1,177 @@
+"""Fetches the metadata of GCP instances and accelerators.
+
+This script fetches the metadata of GCP from a public GitHub repository
+and saves them as CSV files. The metadata include all the information
+except the (on-demand and spot) prices.
+
+NOTE: The TPU metadata is not fetched by this script. It is manually
+maintained by the SkyPilot developers.
+"""
+import os
+from typing import List, Tuple
+
+import pandas as pd
+
+# We rely on the data from https://github.com/Cyclenerd/google-cloud-pricing-cost-calculator  # pylint: disable=line-too-long
+# TODO(woosuk): use the official GCP APIs to fetch the data.
+VM_CSV_URL = 'https://raw.githubusercontent.com/Cyclenerd/google-cloud-pricing-cost-calculator/master/tools/machinetypezone.csv'  # pylint: disable=line-too-long
+GPU_CSV_URL = 'https://raw.githubusercontent.com/Cyclenerd/google-cloud-pricing-cost-calculator/master/tools/acceleratortypezone.csv'  # pylint: disable=line-too-long
```

Comment:
should we point to a version tag of their repo (e.g., https://github.com/Cyclenerd/google-cloud-pricing-cost-calculator/tree/v1.2.0) instead of `master` to avoid any potential change in their future version?

Reply:
If so, we cannot the get up-to-date list of VM instances. As the metadata will be updated manually, I think this is fine.